monkestation/code/modules/antagonists/clock_cult/machines/clockwork_operating_computer.dm

```diff
@@ -13,6 +13,7 @@
 												/datum/surgery/healing/brute/upgraded/femto,
 												/datum/surgery/healing/burn/upgraded/femto,
 												/datum/surgery/healing/combo/upgraded/femto,
+												/datum/surgery/robot_healing/experimental,
```

Comment:
add monkestation edit comments

Reply:
Would this one need a monkestation edit if its already modularized into the monkestation folder?